.storybook/4.SHADOW.stories.mdx

```diff
@@ -144,8 +144,8 @@ As well as the neutral colors for shadow 2 other colors exist that are used for
 | Color       | CSS                           |
 | ----------- | ----------------------------- |
 | **neutral** | `var(--color-shadow-default)` |
-| **primary** | `var(--color-primary-shadow)` |
-| **danger**  | `var(--color-error-shadow)`   |
+| **primary** | `var(--color-shadow-primary)` |
+| **danger**  | `var(--color-shadow-error)`   |
```

Comment:
And here
```suggestion
| **primary** | `var(--color-primary-shadow)` |
| **danger**  | `var(--color-error-shadow)`
```

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/25125/commits/ce1b375551c85b72a97d54226d61f7c3d4b4f884